src/main/java/net/rubygrapefruit/platform/file/FileWatcher.java

```diff
@@ -8,9 +8,9 @@
  * A handle for watching file system locations.
  */
 public interface FileWatcher extends Closeable {
-    void startWatching(File path);
+    void startWatching(File... paths);
```

Comment:
Can we use a collection instead of a varargs argument here? I think that would be better.

Reply:
I actually reverted from using a collection to varargs. It allows for simpler (=unmodified) code in our tests. I don't think for actual applications there is a big difference either way.

Reply:
I think there is, since we will most likely have a collection there, and then then we will need to convert the collection to an array of `File`s, then the array of `File`s will need to be converted to an array of `String`s and then passed to the native side.

Reply:
Not a hill I'm willing to die on. :) Reverted to using collections.